src/openzaak/components/zaken/models/zaken.py

```diff
@@ -552,6 +552,25 @@ class RelevanteZaakRelatie(models.Model):
         ),
     )
 
+    overige_relatie = models.CharField(
+        max_length=100,
+        verbose_name=_("overige relatie"),
+        blank=True,
+        help_text=mark_experimental(
+            "Naam van de overige relatie. Verplicht bij relatie aard `overige`."
```

Comment:
Missed this on the previous review
```suggestion
            "Naam van de overige relatie. Verplicht bij relatie aard `overig`."
```